01-Login/src/components/Callback.vue

```diff
@@ -8,11 +8,7 @@
 export default {
   methods: {
     handleLoginEvent(data) {
-      if (data.state) {
-        this.$router.push(data.state.target || "/");
-      } else {
-        this.$router.push("/");
-      }
+      this.$router.push(data.target || "/");
```

Comment:
shouldn't this be `data.state.target`?
```suggestion
      this.$router.push(data.state.target || "/");
```